doc/protocol.md

```diff
@@ -1025,6 +1025,9 @@ The following options can be specified in the "open" control message:
 
  * "path": The path name of the file to read.
 
+The ready message contains a "size-hint" when the channel is opened with the
```

Comment:
The text wrapping here looks wrong vis-a-vis surrounding text.

Reply:
Well I use vim's formatting which disagrees on the width I suppose? 76 versus 70.. weird!